src/Rapidez.php

```diff
@@ -6,8 +6,30 @@ class Rapidez
 {
     public function config(string $path, $default = null): ?string
     {
-        $configModel = config('rapidez.models.config');
+        return config('rapidez.models.config')::getCachedByPath($path, $default);
+    }
+
+    public function parseWidgetContent($content)
+    {
+        $vars = $this->getWidgetVars($content);
+        if (!isset($vars['type'])) {
+            return '';
+        }
+
+        $parseClass = config('rapidez.widgets.'.$vars['type']);
+        $content = (new $parseClass($vars))->render();
```

Comment:
Maybe check if the class exists? Otherwise a message which is only visible on non-production environments about a non-implemented widget?